contents/blog/ga4-alternatives.md

```diff
@@ -578,3 +578,91 @@ Unsurprisingly, Piwik PRO's roots in Matomo make it a popular choice for users w
+        	<td className="w-3/12 text-center"><strong>Counter</strong></td>
+    	</tr>
+	</thead>
+	<tbody>
+		<tr>
+        	<td>Website analytics</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+    	</tr>
+		<tr>
+        	<td>Funnels</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>User Paths</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td>
+          <td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>Retention</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+       	<td>Revenue tracking</td>
+      		<td className="text-center"><span className="text-green text-lg">✔</span></td>
+       	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>User profiles</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+   		</tr>
+    	<tr>
+        	<td>Cohort analysis</td>
+        	<td className="text-center"><span className="text-green text-lg">✔</span></td> 
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>A/B testing</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>Session recording</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+    	<tr>
+        	<td>Feature flags</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+		<tr>
+        	<td>Self-hosting</td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+        	<td className="text-center"><span className="text-red text-lg">✖</span></td>
+    	</tr>
+	</tbody>
+</table>
+</div>
+
+#### Verdict
+
+Counter is a great choice if you have basic needs. It does most of the things the popular privacy-first analytics tools offer, but does so for free. The simple interface is easy to understand, though it doesn't allow you to drill down as deeply into individual page performance as Fathom or Plausible.
```

Comment:
```suggestion
Counter is a great choice if you have basic needs. It does most of the things the popular privacy-first analytics tools offer, but does so for free. The simple interface is easy to understand, though it doesn't allow you to drill down deeply into individual page performance like Fathom or Plausible.
```
I think this makes this sentence likely more readable, but feel free to disagree.